src/test/test.js

```diff
@@ -2369,11 +2383,16 @@ describe('Floccus', function() {
                       new Folder({title: 'foo',
                         children: [
                           new Bookmark({title: 'url', url: 'http://ur.l/'}),
-                          new Bookmark({title: '-----', url: 'https://separator.floccus.org/?id=467366'}),
+                          new Bookmark({title: '⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯', url: 'https://separator.floccus.org/index.html?id=467366'}),
                           new Bookmark({title: 'url2',url: 'http://ur2.l'}),
-                          new Bookmark({title: '-----', url: 'https://separator.floccus.org/?id=731368'})
-                        ]}),
+                          new Bookmark({title: '⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯', url: 'https://separator.floccus.org/index.html?id=731368'})
+                        ]})                      
                     ]}),
+                    new Folder({title: 'Bookmarks Bar',
+                      children: [
+                        new Bookmark({title: 'url3',url: 'http://ur3.l'}),
+                        new Bookmark({title: '', url: 'https://separator.floccus.org/vertical.html?id=000000'})
```

Comment:
I am not sure how to determine the right ID for expected URL. 
@marcelklehr, could you please advise?

Reply:
Current comment after PR update: https://github.com/floccusaddon/floccus/pull/1846/files#r1932738230